servicebroker/test/routes/binding_test.js

```diff
@@ -176,6 +150,11 @@ describe('binding RESTful API', function() {
         });
       });
       context('when bindingId are conflict with an existed one', function() {
+        beforeEach(function(done) {
+          serviceInstance.findOrCreate(service_condition2).then(function(result) {
```

Comment:
using findOrCreate bothers me. It would seem we don't know if it should already be there or not.
If we expect it not to be there, we should use create()
We also don't need done here.